components/03-modules/slider/_slider.scss

```diff
@@ -79,4 +80,37 @@ $slider__icon-fill-hover           : darken($color-primary, 10%) !default;
         transition: $slider__transition;
         cursor: pointer;
     }
+
+    &__dot-navigation {
+        display: flex;
+        flex-wrap: nowrap;
+        justify-content: center;
+        margin-top: $spacer;
+    }
+
+    &__dot {
+        display: block;
+        padding: $slider__dot-padding;
+        margin-right: $slider__dot-margin;
+        cursor: pointer;
+
+        &--active {
+            & .slider__dot-inside {
```

Comment:
useless space after `&`

Reply:
not space can be useless b/c it's nested element so this space is required in this case. I can delete `&` sign to working fine too

Reply:
ok so `&` is useless

Reply:
Ok I see you fixed it.